client.go

```diff
@@ -423,3 +427,23 @@ func (c *Client) String() string {
 func (c *Client) setHeader(key, val string) {
 	c.headers[key] = val
 }
+
+// Feed opens an event feed from the event source
+func (c *Client) Feed(stream EventSource, opts ...FeedOptFn) (*EventFeed, error) {
+	return newEventFeed(c, stream, opts...)
+}
+
+// FeedFromQuery opens an event feed from a query
+func (c *Client) FeedFromQuery(query *Query, opts ...FeedOptFn) (*EventFeed, error) {
+	res, err := c.Query(query)
```

Comment:
We should validate that the cursor option was not provided here.

Reply:
added a test to validate this case